Source/ENS/Private/Characters/Player/EnsPlayerController.cpp

```diff
@@ -22,6 +25,8 @@ AEnsPlayerController::AEnsPlayerController()
 void AEnsPlayerController::BeginPlay()
 {
 	Super::BeginPlay();
+	ActiveCamera = Cast<UCameraComponent>(GetPawn()->GetComponentByClass(UCameraComponent::StaticClass()));
```

Comment:
we can add a GetCameraComponent in the character, which returns CameraComponent and use it.